app/assets/stylesheets/application.scss

```diff
@@ -65,3 +65,19 @@ $pagination-link-btn-border-color: #ddd;
     }
   }
 }
+
+.profile-head h5{
+    color: #333;
```

Comment:
Store colors in a variable and use that. Like $pagination-link-btn-border-color.

Reply:
Oh sorry, I am not using it anymore. Deleted